LICENSE-binary

```diff
@@ -376,7 +376,7 @@ hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/io/com
 hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/fuse-dfs/util/tree.h
 hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/native/container-executor/impl/compat/{fstatat|openat|unlinkat}.h
 
-com.github.luben:zstd-jni:1.4.9-1
```

Comment:
where does this change come from? a transient import of kafka?

Reply:
Updated LICENSE-binary based on the following result.
`$mvn dependency:tree -pl hadoop-tools/hadoop-kafka
...
[INFO] +- org.apache.kafka:kafka-clients:jar:3.4.0:compile
[INFO] |  +- com.github.luben:zstd-jni:jar:1.5.2-1:runtime
[INFO] |  \- org.lz4:lz4-java:jar:1.7.1:runtime
`
it's right, org.lz4:lz4-java does not require updates.